docker-compose.yml

```diff
@@ -109,25 +109,25 @@ services:
       - AUTH0_M2M_CLIENT_SECRET=updk4Gi1f6ncXDCDBH5ZclbsbIUaZmqvEXYCQLCFI56RnWlnTQXCQe6-h9n86QTv
 
 
-  balrogreactui:
-    build: ./react-ui
-    depends_on:
-      balrogadmin:
-        condition: service_healthy
-    ports:
-      - "9000:9000"
-    environment:
-      - BALROG_ROOT_URL=https://localhost:8010
-      - HOST=0.0.0.0
-      - PORT=9000
-    volumes:
-      - ./react-ui:/app
-      - node_modules:/app/node_modules
-    entrypoint:
-      - /bin/bash
-      - --login
-      - -c
-      - yarn install && yarn start
+#  balrogreactui:
```

Comment:
When you land this PR, if these aren't used, it'd be :ok_hand: if you deleted these rather than just commenting them out (since we have VCS etc etc)

Reply:
Whoops, didn't mean to include this!